File: Beecrowd/Beginner/1101 - 1190/1160 - Population Increase/C++/1160_PopulationIncrease.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int testCases, pa, pb;
    double g1, g2;
    cin >> testCases;

    while (testCases--) {
        cin >> pa >> pb >> g1 >> g2;

        int years = 0;

        while (pa <= pb) {
            pa += pa * (g1 / 100);
            pb += pb * (g2 / 100);
            years++;

            if (years > 100) {
                cout << "Mais de 1 seculo." << endl;
                break;
            }
        }

        if (years <= 100) cout << years << " anos." << endl;
    }
    
    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1160 - Population Increase/Python/1160_PopulationIncrease.py
testCases = int(input())

for i in range(testCases):
    pa, pb, g1, g2 = map(float, input().split())
    
    pa = int(pa)
    pb = int(pb)
    years = 0
    
    while pa <= pb:
        pa += int(pa * (g1 / 100))
        pb += int(pb * (g2 / 100))
        years += 1
        
        if years > 100:
            print("Mais de 1 seculo.")
            break
        
    if years <= 100:
        print(f"{years} anos.")
